pkg/blueprint/customizations.go

```diff
@@ -425,3 +427,27 @@ func (c *Customizations) GetRHSM() *RHSMCustomization {
 	}
 	return c.RHSM
 }
+
+func (c *Customizations) CheckCACerts() error {
+	if c == nil {
+		return nil
+	}
+
+	if c.CACerts == nil {
```

Comment:
This should be `!=`, no? If so ideally that would have triggered a test failure :/

Reply:
Correct, thanks.